### ISSUE TITLE
Unsupported feature http://xmlpull.org/v1/doc/features.html#indent-output

### ISSUE THREAD

**Issue by grzechu92** [Banno/Gordon#6]
Hello :)

While trying to perform some tests everything works fine until it tries to generate some reports. I'm constantly getting:

```
> Task :app:gordonMock FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:gordonMock'.
> unsupported feature http://xmlpull.org/v1/doc/features.html#indent-output
```

Good to mention that issue exists only in our project. Your sample on the same configuration works flawless. I have no idea what causes this issue. Creating my fork without [this line](https://github.com/Banno/Gordon/blob/4fb73afc9b9ee318fd67c26c4c96a04d2b69bad4/gordon-plugin/src/main/kotlin/com/banno/gordon/Xml.kt#L10) solves it.

Do you have any ideas or tips where to look in the issue?
Is some dependency conflict possible?

Regards,
Greg

**Comment by joshschriever** [comment]
That's strange. I know there are different implementations of xmlpull. What operating system are you running?

Could you try replacing [that line](https://github.com/Banno/Gordon/blob/4fb73afc9b9ee318fd67c26c4c96a04d2b69bad4/gordon-plugin/src/main/kotlin/com/banno/gordon/Xml.kt#L10) with the following, and see if that works for you?
```
setProperty("http://xmlpull.org/v1/doc/properties.html#serializer-indentation", "  ")
setProperty("http://xmlpull.org/v1/doc/properties.html#serializer-line-separator", "\n")
```

Thanks!

**Comment by GrzegorzBorkowski-DAZN** [comment]
`BUILD SUCCESSFUL` :)

Issue existed on both MacOS machines and Ubuntu (Dockerized). I was looking for any conflicting dependencies of `xmlpull`, but version is consistent across both your sample and our project... it looks like issue is somewhere else, I have no idea :( 

Do you plan to implement that change? 
Thank you!

**Comment by grzechu92** [comment]
`BUILD SUCCESSFUL` :)

Issue existed on both MacOS machines and Ubuntu (Dockerized). I was looking for any conflicting dependencies of `xmlpull`, but version is consistent across both your sample and our project... it looks like issue is somewhere else, I have no idea :(

Do you plan to implement that change?
Thank you!

**Comment by joshschriever** [comment]
Fixed in [version 1.0.3](https://github.com/Banno/Gordon/releases/tag/1.0.3)
